contracts/contracts/MessageProcessor.sol

```diff
@@ -150,7 +150,7 @@ contract MessageProcessor is Ownable, SnarkCommon, Hasher, CommonUtilities, IMes
     // get the message batch size from the message tree subdepth
     // get the number of signups
     (uint256 numSignUps, uint256 numMessages) = poll.numSignUpsAndMessages();
-    IMACI maci = poll.extContracts();
+    IMACI maci = poll.getMaci();
```

Comment:
I think we could even include this as a immutable variable and set it in the constructor of the Mp contract so that we avoid making an external call every time